source/read/change-streams.txt

```diff
@@ -0,0 +1,261 @@
+array:
+
+- ``$addFields`` or ``$set``: Adds new fields to documents
+- ``$match``: Filters the documents
+- ``$project``: Projects a subset of the document fields
+- ``$replaceWith`` or ``$replaceRoot``: Replaces the input document with the
+  specified document
+- ``$redact``: Restricts the contents of the documents
+- ``$unset``: Removes fields from documents
+
+The following passes a pipeline that includes the ``$match`` stage to the ``watch()``
+method. This instructs the ``watch()`` method to output events only when update
+operations occur:
+
+.. literalinclude:: /includes/read/change-streams.php
+   :start-after: start-change-stream-pipeline
+   :end-before: end-change-stream-pipeline
+   :language: php
+   :dedent:
+
+Modify ``watch()`` Behavior
+---------------------------
+
+To modify the behavior of the ``watch()`` method, you can pass an options array
+as a parameter to ``watch()``. The following table describes useful options you
+can set in the array:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``fullDocument``
+     - | Specifies whether to show the full document after the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see the :ref:`php-change-stream-pre-post-image` section of this
+         guide.
+
+   * - ``fullDocumentBeforeChange``
+     - | Specifies whether to show the full document as it was before the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see :ref:`php-change-stream-pre-post-image`.
+    
+   * - ``resumeAfter``
+     - | Instructs ``watch()`` to resume returning changes after the
+         operation specified in the resume token.
+       | Each change stream event document includes a resume token as the ``_id``
+         field. Pass the entire ``_id`` field of the change event document that
+         represents the operation you want to resume after.
+       | This option is mutually exclusive with ``startAfter`` and ``startAtOperationTime``.
+
+   * - ``startAfter``
+     - | Instructs ``watch()`` to start a new change stream after the
+         operation specified in the resume token. This field allows notifications to
+         resume after an invalidate event.
+       | Each change stream event document includes a resume token as the ``_id``
+         field. Pass the entire ``_id`` field of the change event document that
+         represents the operation you want to resume after.
+       | This option is mutually exclusive with ``resumeAfter`` and ``startAtOperationTime``.
+
+   * - ``startAtOperationTime``
+     - | Instructs the change stream to only provide changes that occurred at or after
+         the specified timestamp. 
+       | This option is mutually exclusive with ``startAfter`` and ``resumeAfter``.
+
+   * - ``typeMap``
```

Comment:
This also seems like something we can omit from this page if the table is admittedly a subset of "useful options". It isn't demonstrated in any examples, and it's already covered in the API reference.